src/routes/docs/products/databases/+layout.svelte

```diff
@@ -89,6 +89,10 @@
                     label: 'CSV imports',
                     href: '/docs/products/databases/csv-imports',
                     new: isNewUntil('31 Jul 2025')
+                },
+                {
+                    label: 'Custom timestamps',
+                    href: '/docs/products/databases/custom-timestamps'
```

Comment:
Should we title it "Timestamp Overrides" considering:

* https://appwrite.io/changelog/entry/2025-08-13
* https://appwrite.io/blog/post/announcing-timestamp-overrides